### ISSUE TITLE
Add integration to HACS

### ISSUE THREAD

**Issue by andreondra** [andreondra/homeassistant-sermatec-inverter#10]
Depends on #6.

**Comment by mathieupotier** [comment]
Following this doc, but I cannot consider this custom_component to be in stable lifecycle... better to keep working a little bit on issues.

https://hacs.xyz/docs/publish/include

**Comment by andreondra** [comment]
I agree, this custom component is in an early alpha stage, I'd like to publish the component when we have all the basic functionality reliably implemented. Also, the logo requirement (#6) could be a little tricky to fulfill, because there could be licensing issues.

**Comment by andreondra** [comment]
I agree, this custom component is in an early alpha stage, I'd like to publish the component when we have all the basic functionality reliably implemented. Also, the logo requirement (#6) could be a little tricky to fulfill, because there could be licensing issues.

**Comment by mathieupotier** [comment]
Sorry I can't help about the logo ... It's not my best skill ... ^_^

But I assume that we can use the [Sermatec logo](https://eur.sermatec-cloud.com/static/favicon.ico) ... but I'm not sure about the legal part ...

**Comment by mathieupotier** [comment]
I'll try to generate a logo with some AI (stable diffusion) got few nice results in another project...
Or maybe the logo of this GitHub organisation... seems clear and straightforward to the point 😅

**Comment by andreondra** [comment]
Well, that's a good idea. You can try to generate some logos, if you get an interesting one, you can send it here and we'll decide what should we use 😁

**Comment by mathieupotier** [comment]
Here's few tries ...

![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/04d7aa2c-e08d-4655-87ef-6655746451b1)

![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/98694c08-3486-472a-809c-5ba757974ffa)

![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/96e134c5-fac5-45ec-9f6b-48b80f76da1e)

![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/2af50e51-75b9-46fe-984b-8d21897f5a48)

![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/e20c8c94-cea5-4210-8e3e-d32a039c4e66)

![image](https://github.com/sermatec-opensource/homeassistant-sermatec-inverter/assets/10141454/4c4277ce-6cd3-43c0-8f5e-6f25c7257e3e)

**Comment by andreondra** [comment]
Wow, these look very good. I like the first, fourth and fifth the most. Maybe we can pick one of those and use them as a organization logo as well?

**Comment by mathieupotier** [comment]
I like the forth and fifth the most too...
I'll republish them later with alpha and proper cropping...

**Comment by andreondra** [comment]
Thanks! :D

**Comment by mathieupotier** [comment]
![Logo4-with-alpha](https://github.com/user-attachments/assets/06b683eb-d795-49ed-bdbc-343a7f2d4f80)

**Comment by mathieupotier** [comment]
![Logo5-with-alpha](https://github.com/user-attachments/assets/3c3a24dc-95aa-4389-88bb-30c0c88bea15)

**Comment by andreondra** [comment]
The second one (with the house) is probably the best. What do you think?

**Comment by andreondra** [comment]
I cleaned up the artifacts and added a little bit of smoothing.

![org_logo](https://github.com/user-attachments/assets/97d642ab-c871-4d95-84c1-3d15551da570)

**Comment by mathieupotier** [comment]
seems the most similar to the one we already have in orga ... suits well !! ^_^
